docs/application/dotnet/guides/user-interface/nui/glview.md

```diff
@@ -0,0 +1,334 @@
+---
+keyword: GLView, callback, configuration, render, GLRenderFrame, control, view, OpenGL, surface, GLES, version, Triangle, thread, method, OpenTK, usage, guide, opentk.tizen
+---
+
+# GLView
+
+GLView is a common component that is used to render the view with OpenGL&reg; ES.
+
+**Figure: GLView rendering a triangle with OpenGL&reg; ES**
+
+![GLView](./media/GLView.png)
+
+
+## Add namespace
+To use GLView, include `Tizen.NUI.Components`  and `OpenTK.Graphics` namespace in your application. You should refer to the `OpenTK` usage:
+
+```csharp
+using Tizen.NUI;
+using Tizen.NUI.Components;
+using OpenTK.Graphics.ES20;
```

Comment:
why don't you use ES30 instead of ES20? ES20 is quite old version.

Reply:
The reason why I chose ES20 is that it is the mandatory version of OpenGL ES.
I've checked the android page today, It uses GLES2 on the page. https://developer.android.com/reference/android/opengl/GLSurfaceView

If you would like to use GLES3, I will change it.